index.html

```diff
@@ -34,5 +34,175 @@ <h1 class="greetings"> Hey there <br>
+                </div>
+              </div>
+               <div class="section6">
+                <article class="quote">
+                  <div class="holder-text">
+                    <img src="images/holder.svg" alt="holder">
+                    <div class="holder-text1">
+                      <img src="images/ellipse.svg" alt="holder">
+                      <ul>
+                        <li class="Language">
+                          <p>Languages</p>
+                          </li>
+                            <ul class="media">
+                              <li>JavaScript</li>
+                              <li>Ruby</li>
+                              <li>HTML</li>
+                              <li>CSS</li>
+                            </ul>
+                      </ul>              
+                    </div>
+                  </div>
+                  </article>
+                    <section class="quote">
+                        <div class="holder-text">
+                        <img src="images/holder.svg" alt="holder">
+                        <div class="holder-text1">
+                          <img src="images/ellipse2.svg" alt="holder">
+                          <ul>
+                            <li class="Language">
+                              <p>Frameworks</p>
+                            </li>
+                            <ul class="media">
+                              <li>React.js</li>
+                              <li>Ruby on Rails</li>
+                              <li>RSpec</li>
+                              <li>Capybara</li>
+                              <li>Selenium</li>
+                            </ul>
+                          </ul>                   
+                          </div>
+                       </div>
+                        </section>
+                        <article class="quote">
+                            <div class="holder-text">
+                            <img src="images/holder.svg" alt="holder">
+                            <div class="holder-text1">
+                              <img src="images/skills.svg" alt="holder">
+                              <ul>
+                                <li class="Language">
+                                  <p>Skills</p>
+                                </li>
+                                <ul class="media">                                                  
+                                  <li>Database Management</li>
+                                  <li>Version Controle</li>
+                                  <li>CLI</li>
+                                  <li>WEb Development</li>
+                                  <li>API Design</li>
+                                <ul>
+                              </ul>
+                             
+                              </div>
+                            </div>
+                        </article>
+                      </ul>
+                    </div>
```

Comment:
- [ ] Please note that the content of this section wit the `about-container` class should be aligned with Flexbox. This will make it responsive on both small & large screens 🤭